### ISSUE TITLE
Add example with `ibmq_token`

### ISSUE THREAD

**Issue by knsmith** [Infleqtion/client-superstaq#798]
### What is happening?

Add or edit existing tutorial to show how to submit to IBM using a personal `ibmq_token`.

**Comment by bharat-thotakura** [comment]
Adding an additional cell here: https://superstaq.readthedocs.io/en/latest/get_started/basics/basics_qss.html#Submit-your-circuit-and-view-results

**Comment by knsmith** [comment]
Awesome! Sounds great. 

Ideas for what we should include? Perhaps just a simple circuit submission to a simulator?

**Comment by bharat-thotakura** [comment]
Yeah submission to a simulator and highlight something like `user_ibmq_token = "<paste token from IBM>"`, optional `user_instance = "<specify token>" or None` if they want to run the tutorial

**Comment by dowusu-antwi** [comment]
This would still be good to add. Add BYOT for IBM to an existing Jupyter notebook (check daily integration test for example)
